src/app/core/dashboard-shortcut-widget/dashboard-shortcut-widget/dashboard-shortcut-widget.component.html

```diff
@@ -17,6 +17,7 @@
         *ngFor="let entry of shortcuts"
         class="dashboard-table-row clickable"
         [routerLink]="[entry.link]"
+        angulartics2On="click" angularticsCategory="Dashboard" angularticsAction="shortcut-widget_click"
```

Comment:
```suggestion
        angulartics2On="click" angularticsCategory="Dashboard" angularticsAction="shortcut-widget_click" [angularticslabel]="entry.link"
```
To also see which shortcuts are used and which ones not.